README.md

```diff
@@ -1,167 +1,87 @@
 
-[NVTabular](https://github.com/NVIDIA/NVTabular), a component of [**NVIDIA Merlin Open Beta**](https://developer.nvidia.com/nvidia-merlin), is a feature engineering and preprocessing library for tabular data that is designed to quickly and easily manipulate terabyte scale datasets and train deep learning (DL) based recommender systems. It provides a high level abstraction to simplify code and accelerates computation on the GPU using the [RAPIDS Dask-cuDF](https://github.com/rapidsai/cudf/tree/main/python/dask_cudf) library.
+The RAPIDS Dask-cuDF library is designed to be interoperable with both PyTorch and TensorFlow using dataloaders that we have developed as extensions of native framework code. NVTabular provides the option to shuffle data during preprocessing, allowing the dataloader to load large contiguous chunks from files rather than individual elements. This allows us to do individual epoch shuffles orders faster than a full shuffle of the datasent. Loading tabular data is almost always a training bottleneck. In our benchmarking, we've seen 10x improvements in training time on the GPU relative to the native dataloaders.
 
-Recommender systems require massive datasets to train, particularly for deep learning based solutions. The transformation of these datasets in order to prepare them for model training is particularly challenging. Often the time taken to complete the necessary steps, such as feature engineering, categorical encoding, and normalization of continuous variables exceeds the time it takes to train a model.
+NVTabular is a component of [**NVIDIA Merlin Open Beta**](https://developer.nvidia.com/nvidia-merlin). NVIDIA Merlin is used for building large-scale deep learning recommender systems, which require massive datasets to train, particularly for deep learning based solutions. The transformation of these datasets in order to prepare them for model training can be challenging. Often the time taken to complete the necessary steps, such as feature engineering, categorical encoding, and normalization of continuous variables, exceeds the time it takes to train a model. With NVTabular being a part of the Merlin ecosystem, it also works with the other Merlin components including [HugeCTR](https://github.com/NVIDIA/HugeCTR) and [Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) to provide end-to-end acceleration of recommender systems on the GPU. Extending beyond model training, with NVIDIA’s Triton Inference Server, the feature engineering and preprocessing steps performed on the data during training can be automatically applied to incoming data during inference.
```

Comment:
This section reads a little weird: the transition from Merlin, to dataset transformation challenges, and then back to Merlin seems a little muddled to me.

Can we just move the dataset transformation challenges sentences out - or even maybe just delete them?

```suggestion
NVTabular is a component of [**NVIDIA Merlin Open Beta**](https://developer.nvidia.com/nvidia-merlin). NVIDIA Merlin is used for building large-scale deep learning recommender systems, which require massive datasets to train, particularly for deep learning based solutions. With NVTabular being a part of the Merlin ecosystem, it also works with the other Merlin components including [HugeCTR](https://github.com/NVIDIA/HugeCTR) and [Triton Inference Server](https://github.com/NVIDIA/tensorrt-inference-server) to provide end-to-end acceleration of recommender systems on the GPU. Extending beyond model training, with NVIDIA’s Triton Inference Server, the feature engineering and preprocessing steps performed on the data during training can be automatically applied to incoming data during inference.
```